docs/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_s3.rst

```diff
@@ -3,16 +3,120 @@
 How to host and share Data Docs on S3
 =====================================
 
-.. admonition:: Admonition from Mr. Dickens
+This guide will explain how to host and share Data Docs on AWS S3.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
 
+    - :ref:`Set up a working deployment of Great Expectations <getting_started>`
+    - `Set up the AWS Command Line Interface <https://aws.amazon.com/cli/>`_
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+Steps
+-----
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+1. Configure an S3 bucket using the AWS CLI. Make sure you modify the bucket name and region for your situation.
 
-If you want to be a real hero, we'd welcome a pull request. Please see :ref:`the Contributing tutorial <tutorials__contributing>` and :ref:`How to write a how to guide` to get started.
+  .. code-block:: bash
+
+    > aws s3api create-bucket --bucket data-docs.my_org --region us-east-1
+    {
+        "Location": "/data-docs.my_org"
+    }
+
+2. Configure your bucket policy to enable appropriate access. The example policy below **enforces IP-based access** - modify the bucket name and IP addresses for your situation. After you have customized the example policy to suit your situation, save it to a file called `ip-policy.json` in your local directory.
+
+  .. admonition:: Important
+  
+      Your policy should provide access only to appropriate users. Data Docs sites can include critical information about raw data and should generally **not** be publicly accessible.
+
+  .. code-block:: json
+  
+      {
+        "Version": "2012-10-17",
+        "Statement": [{
+          "Sid": "Allow only based on source IP",
+          "Effect": "Allow",
+          "Principal": "*",
+          "Action": "s3:GetObject",
+          "Resource": [
+            "arn:aws:s3:::data-docs.my_org",
+            "arn:aws:s3:::data-docs.my_org/*"
+          ],
+          "Condition": {
+            "IpAddress": {
+              "aws:SourceIp": [
+                "192.168.0.1/32",
+                "2001:db8:1234:1234::/64"
+              ]
+            }
+          }
+        }
+        ]
+      }
+
+3. Run the following CLI command to apply the policy:
+
+  .. code-block:: bash
+  
+      > aws s3api put-bucket-policy --bucket data-docs.my_org --policy file://ip-policy.json
+
+4. Add a new S3 site to the ``data_docs_sites`` section of your ``great_expectations.yml`` - you may also replace the default ``local_site``. 
```

Comment:
```suggestion
4. **Add a new S3 site to the data_docs_sites section of your great_expectations.yml**

  Add a new S3 site to the ``data_docs_sites`` section of your ``great_expectations.yml`` - you may also replace the default ``local_site``. 
```

Reply:
Please clarify why I would want to replace the original site vs adding another one. What will happen if I have more than one site in my yml config?